std_msgs/Header header
ComparisonScores[] scores
